src/main/scala/CsvWriter.scala

```diff
@@ -0,0 +1,46 @@
+import com.github.tototoshi.csv.CSVWriter
+
+import scala.collection.immutable.HashMap
+
+object CsvWriter {
+  val outputFilePowerPlants = "./src/main/resources/sources/powerplants.csv"
+  val outputFileGeoStat = "./src/main/resources/sources/geo-stats.csv"
+  def writeHeaderForPowerPlants(outputFile: String, header: List[String] = List("Description", "Answer")): Unit = {
+    val writer = CSVWriter.open(outputFile)
+    writer.writeRow(header)
+  }
+
+  def writePlantsInEachContinent(answer: HashMap[String, Int], outputFile: String = outputFileGeoStat, header: List[String] = List("Continent", "NumberOfPlants")): Unit = {
+    val writer = CSVWriter.open(outputFile)
+    writer.writeRow(header)
+    writer.writeAll(answer.toSeq.map(value => List(value._1, value._2)))
+    writer.close()
+  }
+
+  def writeTotalPower(answer: Double, description: String = "Total capacity of all existing power plants", outputFile: String = outputFilePowerPlants): Unit = {
```

Comment:
Are you sure writting Total Power to file is any different from writing Maximum Number of Plants or Countries with minimum amount of gas stations? I'd say all those functions are 95% copy-paste. Consider refactoring